app/config/env.php

```diff
@@ -82,3 +82,20 @@
     $loader = new Loader\YamlFileLoader($container, new FileLocator(__DIR__ . '/cache_pool'));
     $loader->load($pool . '.yml');
 }
+
+// Yoochoose settings
+if ($value = getenv('YOOCHOOSE_CUSTOMER_ID')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.customer_id', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_LICENSE_KEY')) {
+    $container->setParameter('ez_recommendation.default.yoochoose.license_key', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_SERVER_URI')) {
+    $container->setParameter('ez_recommendation.default.server_uri', $value);
+}
+
+if ($value = getenv('YOOCHOOSE_INCLUDED_CONTENT_TYPES')) {
```

Comment:
@andrerom  [Fixed](https://github.com/kamilmusial/ezstudio-demo/commit/d0e877373f0d12b7d300cf2a5bac2da398dfb3a6) in my fork